check_process

```diff
@@ -1,30 +1,23 @@
-# See here for more information
-# https://github.com/YunoHost/package_check#syntax-check_process-file
-
-# Move this file from check_process.default to check_process when you have filled it.
-
 ;; Test complet
 	; Manifest
-		domain="domain.tld"	(DOMAIN)
-		path="/path"	(PATH)
-		admin="john"	(USER)
+		domain="domain.tld"
+		path="/path"
+		admin="john"
 		language="fr"
+                is_public=1
+                password="1Strong-Password"
 	; Checks
 		pkg_linter=1
-		setup_sub_dir=1
+		setup_sub_dir=0
```

Comment:
Don't we want to enable this test ? (c.f. your post on the forum)

Reply:
Sure, if it's working. :) Right now I can't get it to work.